lib/template-mixins.js

```diff
@@ -76,6 +76,15 @@ module.exports = function (t, fields, options) {
         }
     }
 
+    function pattern(key) {
+        var validation = fields[key] && fields[key].validate || [];
+        var patt = _.findWhere(validation, { type: 'pattern' });
+        if (patt) {
+            return patt.arguments || null;
+        }
+        return null;
+    }
```

Comment:
This idea seems ok in principle, but I think I'd like it to integrate better with the code in https://github.com/UKHomeOffice/passports-form-controller/blob/master/lib/validation/validators.js

In particular the `pattern` validation type does not exist there, but the `regex` type does. There has possibly been some wheel-reinventing along the line somewhere? 

It may also make sense to include some of the validation types that have implicit regexes, such as `alphanum`, `numeric` etc.